highscores/lib.py

```diff
@@ -1,16 +1,22 @@
 from django.http import HttpRequest
 from django.core.mail import send_mail
+from django.utils import timezone
+import logging
 
 from .models import Score, CleanCodeSubmission, ExemptedIP
 from .forms import ScoreForm
 from SRCweb.settings import NEW_AES_KEY, DEBUG, ADMIN_EMAILS, EMAIL_HOST_USER
+import os
 
 from typing import Callable, Union
 from Crypto.Cipher import AES
 from urllib.request import urlopen, Request
+import json
 
 USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'
 
+DISCORD_WEBHOOK_URL = os.getenv("HIGHSCORES_WEBHOOK_URL")
```

Comment:
nit: We currently import/store all our env vars from `SRCweb.settings`

Reply:
Yeah good point, is fixed.